ProfileManager.js

```diff
@@ -115,6 +125,82 @@ export default class ProfileManager {
     return await this._getAgentContent({profileAgentRecord});
   }
 
+  /**
+   * Gets the profile agent assigned to the account associated with the
+   * authenticated session for the profile identified by the given profile ID.
+   *
+   * @param {object} options - The options to use.
+   * @param {string} options.profileId - The ID of the profile to get the
+   *   profile agent for.
+   * @param {string} options.capabilityInvoker - The type of invoker to sign
```

Comment:
This should be `capabilityInvokerType` then ... or `invokerType` if we feel we can shorten it. And, it's not really a `type` actually ... and this gets to a question I have later about using `local` as a "type". Maybe this more of a invocation signing location or something. I think we need a better name for this, it's confusing. Or it should be a flag like `delegateToLocal/delegateLocally=true/false`... but I know that won't square with the `_getInvocationSigner` changes below.

Getting a "local" invocation signer makes sense I think -- but it's a local agent (an agent of the profile agent). So having the difference be between "local" and "agent" is confusing. "Local" and "remote" might work -- with the implication that a local invocation signer has received a delegated zcap. Note that you don't "invoke" locally though -- that is always done remotely. So a "local" capability invoker doesn't make sense. A local signer does.

So this should be `invocationSignerType` and then use "local" or "remote".

Reply:
Do we need to append `Type`? What does it mean for one to specify `remote`? If `remote` is specified, then what entity is signing the invocation... the agent, the profile, something else? It's not clear that specifying `remote` results in using the Agent's signer.

Reply:
Those same questions apply to "local". If you specify "local" which entity is signing? It's not the agent and it's not the profile. It's the profile agent's agent. Maybe we should call it "double agent" :)

I think "local" and "remote" are problematic -- and so is "capabilityInvoker" and "capabilityInvokerType". We're not changing "types" here -- we're changing which entity is signing. The use of "local" and "remote" was intended to refer to where the signature was taking place, but it *also* changes which entity signs (between the profile agent's agent and the agent).

So we still need a better name for this -- and maybe it's a flag not a type. `delegateToLocalEphemeralKey` or `useLocalCapabilityAgent`/`getForLocalCapabilityAgent` are more accurate with respect to what's going on. Maybe `invocationSignerType` needs to be `localCapabilityAgent` or `profileAgent` or `profile`.

Reply:
The original parameter was a flag named `useLocalSigner` which could be updated to `useEphemeralSigner`. Thoughts?

Reply:
`useEphemeralSigner` would be better. Would need to see it in all the places (sorry!). Maybe we want is to say `delegateToEphemeralSigner` or `delegateToLocalSigner`? We're getting a capability ... that we're going to pair up with an invocation signer. So whatever this flag is named is linked to that other concept. I still think "local" isn't quite right -- though I like the terseness of it. What do we want to communicate to the developer about what's going on?

Reply:
I refactored the code to use `useEphemeralSigner`. We want the developer to use the ephemeral signer to sign capability invocations instead of going out to the network to get it signed by some key in a KMS. We will need some text around this to help bolster a developer's understanding of what an "ephemeral signer" is and why it is important.